docs/source/tensors.rst

```diff
@@ -219,6 +219,7 @@ view of a storage and defines numeric operations on it.
    .. automethod:: cpu
    .. automethod:: cross
    .. automethod:: cuda
+   .. automethod:: cummax
```

Comment:
I don't think I saw the doc string for the Tensor version. It should just point to the `torch.` version.